client-src/elements/chromedash-form-field.ts

```diff
@@ -273,9 +281,10 @@ export class ChromedashFormField extends LitElement {
     // form field name can be specified in form-field-spec to match DB field name
     const fieldName = this.fieldProps.name || this.name;
     // choices can be specified in form-field-spec or fetched from API
-    const choices = this.fieldProps.choices || this.componentChoices;
+    const choices: [number, string][] | [number, string, string][] =
+      this.fieldProps.choices || this.componentChoices;
 
-    let fieldHTML = '';
+    let fieldHTML: TemplateResult | string = '';
```

Comment:
I'd probably fix the types by changing the initialization to "html``" instead, since that matches the other assignments.

```suggestion
    let fieldHTML = html``;
```